WebApplication1/Models/Theatre.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace WebApplication1.Models
+{
+    public class Theatre
+    {
+        public int ID { get; set; }
+        [Required]
+        
+        public Movie Movie { get; set; }
+        [Required]
+
+        public Movie[] Movies { get; set; }
```

Comment:
As you can see, this is a start of a many to many relationship, which means since it is a relationship you have to let the framework that this property relates to another model, so when building the database it creates a new table that contains the relationship between theatre and movie.

Also a note, you almost never should use arrays when creating a collection of objects, because as you know, you need to set a limit to the array, instead we use List, Enumerable, Collection, etc. mostly we use List<>

You can learn about configuring a relationship from this link here: https://www.learnentityframeworkcore.com/configuration/one-to-many-relationship-configuration